tracim/tracim/lib/email_fetcher.py

```diff
@@ -188,9 +189,15 @@ def __init__(
         self.token = token
         self.use_html_parsing = use_html_parsing
         self.use_txt_parsing = use_txt_parsing
-        self.lock = filelock.FileLock(filelock_path)
+        self._set_lock(lockfile_path)
         self._is_active = True
 
+    def _set_lock(self, lockfile_path):
+        dir_path = os.path.dirname(lockfile_path)
+        if dir_path:
+            os.makedirs(dir_path, exist_ok=True)
```

Comment:
I would not do this because if you make a typo when writing the file path, it may result in creating some folders that you didn't expect to have. For example, if I write `/home/Damien/tracimcache` instead of `/home/damien/tracim/cache` it will potentially create folders that should not exist.